zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/ZiplinePluginTest.kt

```diff
@@ -60,6 +60,26 @@ class ZiplinePluginTest {
     assertThat(ziplineOut.resolve("basic-lib.zipline").exists()).isTrue()
   }
 
+  /**
+   * Run the Zipline compiler on the Webpack output. It's flattened into a single module and
+   * minified.
+   */
+  @Test
+  fun webpackBuild() {
+    val projectDir = File("src/test/projects/basic")
+
+    val taskName = ":lib:jsBrowserProductionWebpackZipline"
+    val result = createRunner(projectDir, "clean", taskName).build()
+    assertThat(SUCCESS_OUTCOMES)
+      .contains(result.task(taskName)!!.outcome)
+
+    val ziplineOut = projectDir.resolve(
+      "lib/build/distributionsZipline"
+    )
+    assertThat(ziplineOut.resolve(MANIFEST_FILE_NAME).exists()).isTrue()
+    assertThat(ziplineOut.resolve("lib.zipline").exists()).isTrue()
```

Comment:
nit: maybe an assertion that only the 2 files were output (manifest and lib.zipline)

Reply:
new commit added with this change

Reply:
Thanks!